main.py

```
@@ -0,0 +1,609 @@
+            interval=86400,
+            first=10
+        )
+
+        application.job_queue.run_repeating(
+            callback=check_channel_availability,
+            interval=3600,
+            first=30
+        )
+
+        application.job_queue.run_daily(
+            callback=cleanup_expired_subscriptions,
```

Ruff (0.12.2)

Undefined name `cleanup_expired_subscriptions`

(F821)